STEERING-COMMITTEE.md

```diff
@@ -46,6 +48,37 @@ the way we run this committee, based on feedback from the community.
 1. Act as the final escalation point and decider for any disputes, issues,
    clarifications, or escalations within the project scope.
 
+## Knative Trademark Committee Charter
+
+1. Manage licensing the Knative brand and decide which things can be called
+   "Knative" and how that mark can be used in relation to other efforts or
+   vendors.
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
+1. Review and approve definition of Knative Core scope.
```

Comment:
Would suggest us to be specific about what "Core" is. If it's serving, eventing, others - let's be clear about it :)

Reply:
+1, i think adding specificity here will answer a lot of questions

Reply:
Note that this avoids the decision on what "Core" means by allowing KTC to decide if defining a mark around "Knative Core" is useful to the stakeholders.

If such a mark is useful, then I'm all for it. It seems like we've assumed that such a mark must exist, and are now fighting over what it must mean. If we step back to see if it's needed, hopefully a definition will suggest itself at the same time.